target/linux/ath79/dts/qca9533_qca_ap143-16M.dts

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9533_qca_ap143.dtsi"
+
+/ {
+	model = "Qualcomm Atheros AP143 (16M) reference board";
+	compatible = "qca,ap143-16M", "qca,qca9533";
```

Comment:
Please only use lower-case characters for the compatible (and thus, image/DTS/device definition name).